docs/static/azure-module.asciidoc

```diff
@@ -0,0 +1,591 @@
+[role="xpack"]
+[[azure-module]]
+=== Azure Module  [Experimental]
+experimental[]
+
+The https://azure.microsoft.com/en-us/overview/what-is-azure/[Microsoft Azure] module in Logstash helps you easily integrate your Azure
+activity logs and SQL diagnostic logs with the Elastic Stack. 
+
+NOTE: The Logstash Azure module is an
+https://www.elastic.co/products/x-pack[{xpack}] feature under the Basic License
+and is therefore free to use. Please contact
+mailto:monitor-azure@elastic.co[monitor-azure@elastic.co] for questions or more
+information.
+
+The Azure module uses the
+{logstash-ref}/plugins-inputs-azure_event_hubs.html[Logstash Azure Event Hubs
+plugin] to consume data from Azure Event Hubs. The module taps directly into the
+Azure dashboard, parses and indexes events into Elasticsearch, and installs a
+suite of {kib} dashboards to help you start exploring your data immediately.   
+
+[[azure-dashboards]]
+==== Dashboards
+
+These {kib} dashboards are available and ready for you to use. 
+
+* *Overview*. Top-level view into your Azure operations, including info about users, resource groups, service health, access, activities, and alerts. 
+
+* *Alerts*. Alert info, including activity, alert status (activated, resolved, succeeded), and alerts heatmap
+
+* *SQL DB Overview*. Top-level view into your SQL databases, including counts for databases, servers, resource groups, and subscriptions.
+
+* *SQL DB Database View*. Detailed info about each SQL database, including wait time, errors, DTU and storage utilization, size, and read and write input/output.
+
+* *SQL DB Queries*. Info about SQL database queries, including DTU Utilization, errors, and query duration and wait time. 
+
+* *User Activity*. Info about system users, their activity, and requests.
+
+You can use the dashboards they are, or tailor them to meet your needs.
+
+==== Azure_event_hubs plugin
+
+The Azure module uses the `azure_event_hubs` plugin. Basic understanding of the
+plugin is helpful when you set up the Azure module. See
+{logstash-ref}/plugins-inputs-azure_event_hubs.html[azure_event_hubs plugin
+documentation] for more information about configurations and options. 
+
+
+[[azure-prereqs]]
+==== Prerequisites
```

Comment:
We also need to add or link to instructions for setting up Azure Monitor on the Azure side. Azure Monitor centralizes the logs to an Event Hub which we ingest from.

https://docs.microsoft.com/en-us/azure/monitoring-and-diagnostics/monitoring-overview-azure-monitor

Reply:
Specifically: https://docs.microsoft.com/en-us/azure/monitoring-and-diagnostics/monitoring-stream-activity-logs-event-hubs and https://docs.microsoft.com/en-us/azure/sql-database/sql-database-metrics-diag-logging for how to setup on Azure side of things.  "Stream to an event hub" option for the latter.